packages/mapviewer/src/config/staging.config.js

```diff
@@ -71,3 +71,10 @@ export const REPORT_PROBLEM_HOSTNAMES = [
  * @type {String[]}
  */
 export const WHITELISTED_HOSTNAMES = ['test.map.geo.admin.ch', 'map.geo.admin.ch']
+
+/**
+ * List of blocked file extensions
+ *
+ * @type {String[]}
+ */
+export const BLOCKED_EXTENSIONS = ['exe', 'bat', 'cmd', 'sh', 'msi', 'scr', 'vbs', 'dll']
```

Comment:
Could go into a new `security.config.js` file, as this has not much to do with staging. `WHITELISTED_HOSTNAMES` could also be moved into this new file